app/preview/TemplateParams.scala

```diff
@@ -741,6 +741,36 @@ object TemplateParams {
       "receivedDate" -> "20 April 2017", "identifier" -> "12345",
       "subject" -> "subject"
     ),
+    "transactionEngineHMRCPSAPRACAUTHSuccess" -> Map(
+      "receivedTime" -> "21:45", "receivedDate" -> "20 April 2017", "identifier" -> "12345",
+      "subject" -> "subject"
+    ),
+    "transactionEngineHMRCPSAPRACAUTHFailure" -> Map(
+      "receivedTime" -> "21:45", "receivedDate" -> "20 April 2017", "identifier" -> "12345",
+      "subject" -> "subject", "errorType" -> "business",
+      "errorTextBlock" -> "<p>Error Code: 123 Error Message: An Error</p>"
+    ),
+    "transactionEngineHMRCPSAPRACAUTHFailure" -> Map(
+      "receivedTime" -> "21:45", "receivedDate" -> "20 April 2017", "identifier" -> "12345",
+      "subject" -> "subject",
+      "errorType" -> "fatal",
+      "errorNumber" -> "123", "errorMessage" -> "This is an error"
+    ),
+    "transactionEngineHMRCPSAPRACDEAUTHSuccess" -> Map(
+      "receivedTime" -> "21:45", "receivedDate" -> "20 April 2017", "identifier" -> "12345",
+      "subject" -> "subject"
+    ),
+    "transactionEngineHMRCPSAPRACDEAUTHFailure" -> Map(
```

Comment:
Hi @yangwong9664, I understand that you have different templates, but the `exampleParams` value is a `Map[String, Map[...]]` so if you use the same key twice one will be ignored. It's ok to have it but just use different keys in order to retrieve the desired params from the Map.